L1Trigger/L1THGCal/interface/backend/HGCalMulticlusteringHistoImpl.h

```diff
@@ -70,40 +55,22 @@ class HGCalMulticlusteringHistoImpl{
     
     std::vector<std::pair<GlobalPoint, double> > computeThresholdSeeds( const Histogram & histoClusters );
 
-    std::vector<l1t::HGCalMulticluster> clusterSeedMulticluster(const std::vector<edm::Ptr<l1t::HGCalCluster>> & clustersPtrs,
-                                                                const std::vector<std::pair<GlobalPoint, double> > & seeds);
-
-
-    void finalizeClusters(std::vector<l1t::HGCalMulticluster>&,
-            l1t::HGCalMulticlusterBxCollection&,
-            const HGCalTriggerGeometryBase&);
-    
-    double dr_;
-    std::vector<double> dr_byLayer_coefficientA_;
-    std::vector<double> dr_byLayer_coefficientB_;
-    double ptC3dThreshold_;
 
     std::string multiclusterAlgoType_;
-    std::string cluster_association_input_;
     MulticlusterType multiclusteringAlgoType_;
```

Comment:
Could profit from this restructuring to improve a bit the naming. It is more a seeding type than a cluster type.